File: media/src/test/java/com/mparticle/Tester.java
package com.mparticle;

import com.mparticle.events.MediaType;
import com.mparticle.events.StreamType;

public class Tester {


    public void main() {
        MediaSession mediaSession = MediaSession.builder()
                .duration(123)
                .title("asdv ")
                .mediaType(MediaType.AUDIO)
                .streamType(StreamType.LIVE_STEAM)
                .build();
    }
}

File: media/src/test/java/com/mparticle/Tester.kt
package testpckg

import com.mparticle.MediaSession
import com.mparticle.events.MediaType
import com.mparticle.events.StreamType


fun main() {
    val mediaSession = MediaSession.builder {
        title = "hello"
        mediaContentId = "123"
        duration = 1000
        streamType = StreamType.LIVE_STEAM
        mediaType = MediaType.VIDEO
    }

}